front-end/src/components/modal/create/CreateGraphModal.js

```diff
@@ -92,7 +123,8 @@ export const CreateGraphModal = ({ setModal, setChartInformation, setSuccessMess
         }}>Next</button>
       </div>
     </div>,
-    <AnalyzersAndSeries dimensions={dimensions} columns={columns} movePage={movePage} seriesInfo={seriesInfo} setSeriesInfo={setSeriesInfo} setSuccessMessage={setSuccessMessage} setDimensions={setDimensions} graphType={graphType}/>
+
```

Comment:
Get rid of this line, we don't break our HTML up so this looks weird

Reply:
removed line